app/controllers/report_controller.rb

```diff
@@ -178,6 +178,10 @@ def x_show
   def tree_select
     @edit = nil
     @sb[:select_node] = false
+    if @sb.try(:[], :flash_msg)
```

Comment:
FYI above numbers are for manageiq repo, for manageiq-ui-classic the number are:

```$ grep -rn \\.try app/ | wc -l```
330

```$ grep -rn \\.key? app/ | wc -l```
206